backend/lib/plugins/socket.js

```diff
@@ -437,6 +437,19 @@ function onSocketConnect(socket) {
     );
   });
 
+  socket.on("CLEAR_NOTIFICATION", async (data) => {
+    const { userId } = socket;
+    const { notificationId } = data;
+    const notification = await Notification.findById(notificationId);
```

Comment:
This can be combined in one query:

```
await Notification.findOneAndUpdate(
  {
    _id: this.mongo.base.Types.ObjectId(notificationId),
    receiver: this.mongo.base.Types.ObjectId(userId),
  },
  {
    isCleared: true,
  },
);
```